web/templates/settings/profile.html

```diff
@@ -164,4 +168,20 @@ <h3 class="ui header">{{ .T "Playstyle" }}</h3>
       </div>
     </div>
   </div>
+  <script>
+    function validateLeaderboardSizeInput() {
+      const leaderboardInputEl = document.getElementById("lbSizeInput");
+      if (isNaN(leaderboardInputEl.valueAsNumber)) {
+        leaderboardInputEl.value = "";
```

Comment:
API endpoint isn't going to take a null as the database can't be nullable so this should probably be a default value (ideally 100)

Reply:
Ideally yes and that's what I did initially however it will not allow user to clear the input to eg. pick a number from scratch

Reply:
can we block saving until they enter a valid value then?